src/app/components/layout/navbar/navbar.component.ts

```diff
@@ -293,10 +294,6 @@ export class NavbarComponent implements OnInit, AfterViewInit, OnDestroy {
     clearInterval(this.timer);
   }
 
-  ngOnDestroy() {
-    clearInterval(this.timer);
```

Comment:
is this happening via super?

Reply:
ngOnDestroy() revered now at child component level to fix this issue.